schemas/components/schemas/TamperingResult.yaml

```
@@ -16,6 +16,12 @@ properties:
     example: 0
     minimum: 0
     maximum: 1
+  antiDetectBrowser:
+    type: boolean
+    description: >-
+      Flag indicating whether anti-detect browser was detected.
```

Can we better define "anti-detect browser"?

```suggestion
      Is `true` if the identified browser is one of "anti-detect" browsers specifically designed to evade identification and fingerprinting, for example, Octo Browser.
```